ui/src/org/pentaho/di/ui/spoon/DatabasesCollector.java

```diff
@@ -79,10 +79,10 @@ public DatabaseMeta getMetaFor( String dbName ) {
     if ( names2metas == null ) {
       throw exception();
     }
-    return names2metas.get( dbName.toLowerCase() );
+    return names2metas.get( dbName );
```

Comment:
If database meta names can be the same (without regards to case), why are we sorting on case insensitive order?